cmd/minikube/cmd/root.go

```diff
@@ -340,3 +341,13 @@ func addToPath(dir string) {
 func validateUsername(name string) bool {
 	return len(name) <= 60
 }
+
+// applyToAllCommands applies the provided func to all commands including sub commands
+func applyToAllCommands(cmd *cobra.Command, f func(subCmd *cobra.Command)) {
+	for _, c := range cmd.Commands() {
+		f(c)
+		if c.HasSubCommands() {
+			visitAllCommand(c, f)
```

Comment:
```suggestion
			applyToAllCommands(c, f)
```
Sorry, forgot to update this in my suggestion